src/java.security.jgss/share/classes/sun/security/jgss/spnego/SpNegoContext.java

```diff
@@ -874,11 +873,7 @@ private byte[] GSS_initSecContext(byte[] token) throws GSSException {
         }
 
         // pass token
-        if (token != null) {
-            tok = token;
-        } else {
-            tok = new byte[0];
-        }
+        tok = Objects.requireNonNullElseGet(token, () -> new byte[0]);
```

Comment:
Is this how `requireNonNullElseGet` is meant to be used?

Reply:
I think so - return the token, or if the token is null, return an empty byte array.